bucket/meld.json

```
@@ -1,8 +1,9 @@
 {
     "homepage": "http://meldmerge.org/",
-    "version": "3.18.3",
-    "url": "https://download.gnome.org/binaries/win32/meld/3.18/Meld-3.18.3-win32.msi",
-    "hash": "196ab263e5425407609addc8e8de6eeba8ebfca8d53902b7ff43f169bdb8564c",
+    "description": "A visual diff and merge tool",
+    "version": "3.20.0",
+    "url": "https://download.gnome.org/binaries/win32/meld/3.20/Meld-3.20.0-mingw.msi",
+    "hash": "9f36ac1bba647eb53ae20864c02c2f8b17467973fc71e1700abd9b18272bc8cf",
     "bin": [
```

There is no need to rename this bin.

```suggestion
    "bin": "Meld.exe",
```